include/adaflo/level_set_okz_advance_concentration.h

```diff
@@ -113,9 +113,8 @@ class LevelSetOKZSolverAdvanceConcentration
     const LevelSetOKZSolverAdvanceConcentrationBoundaryDescriptor<dim> &boundary,
     const MatrixFree<dim> &                                             matrix_free,
     const LevelSetOKZSolverAdvanceConcentrationParameter &              parameters,
-    double &                                                global_max_velocity,
-    const DiagonalPreconditioner<double> &                  preconditioner,
-    AlignedVector<Tensor<1, dim, VectorizedArray<double>>> &evaluated_convection);
```

Comment:
@mschreter Please not the updated interface.

Reply:
nice.